src/js/components/cycleCount/CycleCountDiscrepancyInfoBar.jsx

```diff
@@ -0,0 +1,30 @@
+import React from 'react';
+
+import PropTypes from 'prop-types';
+
+import Translate from 'utils/Translate';
+
+const CycleCountDiscrepancyInfoBar = ({ outOfStockItems }) => (
```

Comment:
isn't this popup meant to show ANY stock changes (not just ones where QoH==0) that happen when re-fetching/reloading the data? So not just outOfStockItems but itemsWithStockChange (or something).

Also, I think "CycleCountDiscrepancyInfoBar" could be a confusing name because we already use "discrepancy" to mean when QoH != quantity counted, but that's not what we're talking about here. Here it's when QoH has changed after a reload.

CycleCountStockChangeInfoBar or something like that is clearer to me

Reply:
I don't think so, the ticket description says clearly that we want the popup/info bar only if QOH is equal to 0 (removed inventory item during count).
It's not really related to cycle count reload, it's the case when you want to submit a count (go to resolve page) and in the meantime the stock changed to 0.

Reply:
Moreover I think that this popup is supposed to show for QOH==0, because we are supposed to hide such a row in the table, so this popup is not to have users confused that an item suddenly disappeared from the table.

Reply:
> the ticket description says clearly that we want the popup/info bar only if QOH is equal to 0

I read it and yeah you're right. This is about rows disappearing and us wanting to notify the user about it. I still think the naming is confusing though and CycleCountStockChangeInfoBar or even CycleCountStockDiscrepancyInfoBar would be clearer IMO.

> It's not really related to cycle count reload

I'm still not super familiar with how the frontend but won't it also display if you reload and a product is removed?